buildenv/jenkins/JenkinsfileBase

```diff
@@ -12,101 +11,82 @@ def getBuildList() {
 	return TESTPROJECT
 }
 
-def getJCK(){
-	dir("$WORKSPACE/jck/"){
-		def jck_suite = "runtime"
-		def jck_version = "0"
-		if (env.JCK_VERSION == "jck8b") {
-			jck_version = "8b"
-		} else if (env.JCK_VERSION == "jck9") {
-			jck_version = "9"
-		} else if (env.JCK_VERSION == "jck10") {
-			jck_version = "10"
+def archiveTestBinaries() {
+	dir("$WORKSPACE") {
+		sh "tar -zcf test-binaries.tar.gz ./openjdk-tests ./jvmtest ./openjdkbinary"
+	}
+	archiveArtifacts artifacts: '**/test-binaries.tar.gz', fingerprint: true, allowEmptyArchive: false
+}
+
+def stageTestBinaries() {
+	sh "wget -q --no-check-certificate --header 'Cookie: allow-download=1' ${env.BUILD_URL}artifact/test-binaries.tar.gz " 
```

Comment:
Any reason not using Copy Artifact Plugin to get artifacts?

Reply:
Didn't think about it initially, thanks for the suggestion. I will try that plugin on both private and public Jenkins server and verify if there is any issue with that. I will create a separate PR to update the JenkinsfileBase to use Copy Artifact Plugin, which will depend on this change set.